server/src/routes/api/families.js

```diff
@@ -3,6 +3,7 @@ import FamilyProfile from '../../models/FamilyProfile';
 
 export default Router()
   .post('/', (req, res, next) => {
+    // only families and admins should be able to create this profile
```

Comment:
This should be an authenticated route. `user` should not be taken from the `req.body`, it should be taken from the authenticated user `req.user`